db/migrate/20161214155109_add_default_ticket_creation_is_open_to_the_world_to_tenants.rb

```diff
@@ -0,0 +1,15 @@
+class AddDefaultTicketCreationIsOpenToTheWorldToTenants < ActiveRecord::Migration
+  def up
+    if column_exists? :tenants, :ticket_creation_is_open_to_the_world
```

Comment:
`column_exists?` are not needed on the tenants table when changing a column, but just when creating one. For every tenant we run migrations, but we share 1 tenants table. Creating or removing a column twice will result in errors, but changing it will not.